x-pack/plugins/endpoint/common/types.ts

```diff
@@ -328,6 +328,10 @@ export interface EndpointEvent {
     name: string;
     parent?: {
       entity_id: string;
+      parent?: {
```

Comment:
why do we need process.parent.parent?

Reply:
we don't, not sure why I added this. Bad merge or something in the demo data? Removing

Reply:
we don't, not sure how this was added. bad merge or something. removing.